abses/space/nature.py

```
@@ -0,0 +1,107 @@
+
+        Returns:
+            Newly created PatchModule instance.
+
+        Note:
+            The first created module automatically becomes the major layer.
+            The module is automatically added to nature's layers collection.
+        """
+        # Create the module directly with the unified API
+        module = super().create_module(
+            module_cls=module_cls,
+            *args,
```

Ruff (0.14.0)

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)